packages/openswap/src/types.ts

```diff
@@ -71,30 +71,24 @@ export type CompletedOrder = {
 }
 export type CompletedOrderResponse = CompletedOrder[]
 
-export type Protocol =
+export type Provider =
   | 'minswap'
   | 'sundaeswap'
   | 'wingriders'
+  | 'muesliswap'
   | 'muesliswap_v1'
   | 'muesliswap_v2'
   | 'muesliswap_v3'
   | 'muesliswap_v4'
   | 'vyfi'
   | 'spectrum'
+// | 'muesliswap_clp'
```

Comment:
@jorbuedo @lisicky I see this provider often in the requests while inspecting can we check with muesli if they have a list, endpoint some place to get the entire providers.